backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/domain/mapper/ConfirmationCodeMapper.java

```diff
@@ -7,7 +7,7 @@
 import ca.gov.dtsstn.cdcp.api.service.domain.ConfirmationCode;
 import jakarta.annotation.Nullable;
 
-@Mapper
+@Mapper(componentModel = "spring")
```

Comment:
We can remove `componentModel = "spring"` since we're currently using `Mappers.getMapper(..)` to get the `ConfirmationCodeMapper` where needed.